tests/genome/test_clust.py

```
@@ -0,0 +1,64 @@
+        )
+    ) == [1, 2, 3, "4_a", "4_a", "5_a"]
+
+
+def test_unionfind():
+    assert set(
+        frozenset(i) for i in UnionFind.igroup((("a", "b"), ("b", "c"), ("d", "e")))
+    ) == {frozenset({"a", "b", "c"}), frozenset({"d", "e"})}
+
+
+@temp_output
+def test_run_pairwise_ani(test_temp: Path):
```

Ruff (0.11.9)

Redefinition of unused `test_temp` from line 11

(F811)